src/map/ai/controllers/mob_controller.cpp

```diff
@@ -539,7 +539,7 @@ void CMobController::DoCombatTick(time_point tick)
     {
         return;
     }
-    else if (m_Tick >= m_LastMobSkillTime && xirand::GetRandomNumber(10000) <= PMob->TPUseChance() && MobSkill())
+    else if (m_Tick >= m_LastMobSkillTime && xirand::GetRandomNumber(1,10000) <= PMob->TPUseChance() && MobSkill())
```

Comment:
Looks good to me 👍 
Can you ninja in a space between those args? then we should be good to go

Reply:
done